billboard.js

```diff
@@ -0,0 +1,5 @@
+function getBillboardPrice (characters, costPerCharacter) {
+    console.log (characters.length * costPerCharacter)
```

Comment:
The function should return the price, not log it to console. Remember that console.log i only for programmers. The function needs to return a value.